ionic-test-app/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "ionic-test-app",
+  "version": "0.0.1",
+  "author": "Ionic Framework",
+  "homepage": "http://ionicframework.com/",
+  "private": true,
+  "scripts": {
+    "clean": "ionic-app-scripts clean",
+    "build": "ionic-app-scripts build",
+    "lint": "ionic-app-scripts lint",
+    "ionic:build": "ionic-app-scripts build",
+    "ionic:serve": "ionic-app-scripts serve"
+  },
+  "dependencies": {
+    "@angular/animations": "^5.0.2",
+    "@angular/cdk": "^5.0.0-rc.1",
+    "@angular/common": "5.0.0",
+    "@angular/compiler": "5.0.0",
+    "@angular/compiler-cli": "5.0.0",
+    "@angular/core": "5.0.0",
+    "@angular/flex-layout": "^2.0.0-beta.10-4905443",
+    "@angular/forms": "5.0.0",
+    "@angular/http": "5.0.0",
+    "@angular/material": "^5.0.0-rc.1",
+    "@angular/platform-browser": "5.0.0",
+    "@angular/platform-browser-dynamic": "5.0.0",
+    "@ionic-native/core": "4.4.0",
+    "@ionic-native/splash-screen": "4.4.0",
+    "@ionic-native/status-bar": "4.4.0",
+    "@ionic/storage": "2.1.3",
+    "angular2-json-schema-form": "git://github.com/jessehon/angular2-json-schema-form-build.git",
```

Comment:
It may be better to use the relative path with this dependency (see https://stackoverflow.com/a/14387210)